src/Changes.php

```diff
@@ -78,7 +78,7 @@ public function mergeWith(self $other): self
      *
      * @return Traversable<int, Change>
      */
-    public function getIterator(): iterable
+    public function getIterator(): Traversable
```

Comment:
Oh, good point, this could stay iterable! 🤔

Reply:
Checked: this is necessary for PHP 8.1 compatibility, where `IteratorAggregate#getIterator(): Traversable` is declared.

The change is therefore valid, and shouldn't pose a downstream BC break, since `Traversable` is covariant with `iterable` here